languages/python/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,49 @@
-# How to implement an Python concept exercise
+# How to implement a Python concept exercise
 
-TODO: describe how to implement a concept exercise for the Python track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes the steps required to implement a concept exercise for the Python track. As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/docs/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in snake_case (e.g. `anonymous_methods`).
```

Comment:
Did you mean, "the name of the **concept**" here?

Reply:
@paparomeo the same convention will apply to both concept and practice exercises, so the underlying `<SLUG>` is the exercise's name, regardless of which type of exercise it is.

However the line is still wrong, with the wrong case identified for the `<SLUG>` ... these should probably read:

```
<SLUG> : the name of the exercise in kebab-case (e.g. `anonymous-methods`) for use in the directory name
<NAME> : the name of the exercise in snake_case (e.g. `anonymouse_methods`) for use in the implementation files
```